src/main/resources/database/changes/0.0/create_secure-auth.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE SECUR_AUTH.USER (
+    ID BIGINT AUTO_INCREMENT PRIMARY KEY,
+    USER_NAME VARCHAR(50) NOT NULL,
+    PASSWORD VARCHAR(200) NOT NULL,
+    USER_INFO_ID BIGINT NOT NULL,
```

Comment:
I do not see any foreign key created at the databse level. Please create foreign keys wherever is applicable.